components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/utils/WorkflowMappingUtil.java

```diff
@@ -0,0 +1,114 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Year should be 2024 and LLC instead of Inc.

Reply:
Fixed